packages/starknet_provider/lib/src/model/block_with_receipts.dart

```diff
@@ -41,10 +39,10 @@ class BlockWithReceipts with _$BlockWithReceipts {
 @freezed
 class GasPrice with _$GasPrice {
   const factory GasPrice({
-    @JsonKey(name: 'price_in_fri') String? priceInFri,
-    @JsonKey(name: 'price_in_wei') String? priceInWei,
+    String? priceInFri,
+    String? priceInWei,
```

Comment:
Sorry I miss this one in my previous review.
I should be `Felt` here see https://github.com/starkware-libs/starknet-specs/blob/4233164155b8a279b408b9638c34db2602b5cdc0/api/starknet_api_openrpc.json#L3757-L3775